src/arrayMethodSort.js

```diff
@@ -4,9 +4,24 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareF) {
```

Comment:
Good decision to put the default comparing function right to the parameter. I didn't think of that!

Reply:
Thanks)